src/ActorMultiVertex.cpp

```diff
@@ -389,7 +389,7 @@ CubicSplineN* ActorMultiVertex::GetSpline(size_t i)
 	return &(_splines[i]);
 }
 
-void ActorMultiVertex::SetState(size_t i)
+void ActorMultiVertex::SetState(int i)
```

Comment:
There was already a warning which was why I changed it to int. By keeping it as size_t, you run into a warning regarding hiding Actor's virtual SetState function/ I will fix it in a separate PR.